.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: Create release PR or publish GitHub release and Rust crate
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: dtolnay/rust-toolchain@stable
+    - uses: MarcoIeni/release-plz-action@v0.5
```

Comment:
Also, why are we using `release-plz`? I don't think we're looking to publish the rust code?

Reply:
Yeah, it worked well for my package (xdot-rs) because I both published the crate and the python package. but that won’t work here.